PanCardView/Common/CardsView.cs

```diff
@@ -857,10 +858,13 @@ protected virtual void SetSelectedItem()
             SelectedItem = GetItemByIndex(index);
         }
 
-        protected virtual async void AdjustSlideShow(bool isForceStop = false)
+        protected virtual async void AdjustSlideShow(bool? isForceStop = null)
         {
+            bool isNotVisible = !IsVisible || Opacity <= 0;
+            isForceStop ??= isNotVisible || IsUserInteractionRunning || IsAutoInteractionRunning;
```

Comment:
Since we are going to revert back to `isForceStop` be a non-nullable, the code will look like this:

```csharp
isForceStop ||= !IsVisible || Opacity <= 0 || IsUserInteractionRunning || IsAutoInteractionRunning;
```